### PR TITLE
Updated import files and export settings.

### PR DESCRIPTION
**Pull request by Malcolmnixon**
This pull request performs the following changes:
 - Adds ASTC/ETC2 texture format
 - Updates all .import files
 - Adds export settings for known Android headsets
 - Rebuilds OpenXR action map

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 *.TMP
 logs
 Thumbs.db
+android/
```
Comment (BastiaanOlij) [inline .gitignore:12]: Needs a newline at the end

File: assets/ambientcg.com/SkyOnlyHDRI023_2K-TONEMAPPED.jpg.import [hunk 2]
```diff
@@ -18,9 +18,9 @@ dest_files=["res://.godot/imported/SkyOnlyHDRI023_2K-TONEMAPPED.jpg-bcb5464e5ed4
 [params]
 
 compress/mode=2
+compress/high_quality=false
```
Comment (BastiaanOlij) [inline assets/ambientcg.com/SkyOnlyHDRI023_2K-TONEMAPPED.jpg.import:21]: I wonder what happens if we set this to true, seeing quality of the panoramic sky seems so-so

Also double check if mipmaps are off

File: assets/nasa/starmap_and_constellation_figures_4k.jpg.import [hunk 3]
```diff
@@ -18,9 +18,9 @@ dest_files=["res://.godot/imported/starmap_and_constellation_figures_4k.jpg-4444
 [params]
 
 compress/mode=2
+compress/high_quality=false
```
Comment (BastiaanOlij) [inline assets/nasa/starmap_and_constellation_figures_4k.jpg.import:21]: Same here, lets see if high quality makes a difference and make sure mipmaps are off.

**Review by BastiaanOlij — APPROVED** [review]
Looks good to me, we'll see how it goes in practice :)

File: .gitignore (hunk 1 above)
Comment (Malcolmnixon) [reply .gitignore]: Done

File: assets/ambientcg.com/SkyOnlyHDRI023_2K-TONEMAPPED.jpg.import (hunk 2 above)
Comment (Malcolmnixon) [reply assets/ambientcg.com/SkyOnlyHDRI023_2K-TONEMAPPED.jpg.import]: Set to high-quality and disabled mipmap generation.

File: assets/nasa/starmap_and_constellation_figures_4k.jpg.import (hunk 3 above)
Comment (Malcolmnixon) [reply assets/nasa/starmap_and_constellation_figures_4k.jpg.import]: Set to high-quality and disabled mipmap generation.

**Pull request merged**